### PR TITLE
chore: Use HDS components in auth methods list view

### PR DESCRIPTION
**Pull request by DhariniJeeva**
This pr updates auth method list table, dropdown and notification components to HDS

Before: (Rose)

Screenshot 2023-05-11 at 9 26 23 PM

Screenshot 2023-05-11 at 9 26 18 PM

After:(HDS)
 
Screenshot 2023-05-12 at 2 43 17 PM
Screenshot 2023-05-12 at 2 43 22 PM

### CODE REVIEW

**Review by cameronperera — APPROVED** [review]
LGTM

**Review by ZedLi — CHANGES_REQUESTED** [review]

File: ui/admin/app/templates/scopes/scope/auth-methods/index.hbs [hunk 1]
```diff
@@ -93,100 +93,91 @@
...
-                  <LinkTo
-                    @route='scopes.scope.auth-methods.auth-method'
-                    @model={{authMethod.id}}
-                  >
-                    {{authMethod.displayName}}
-                  </LinkTo>
-                {{else}}
-                  {{authMethod.displayName}}
-                {{/if}}
-                <p>{{authMethod.description}}</p>
```
[the remark this answers is not in the document]
Comment (DhariniJeeva) [reply ui/admin/app/templates/scopes/scope/auth-methods/index.hbs:133]: Done! thanks for the catch

**Review by ZedLi — APPROVED** [review]
Looks great!

**Review by lisbet-alvarez — APPROVED** [review]
LGTM!

**Pull request merged**